classification/apply_HLDA.m

```diff
@@ -0,0 +1,61 @@
+function out = apply_HLDA(epo, ival, C)

```

Comment:
If the apply function has arguments other than (fv, C) -- such as 'ival' here -- they cannot be used in crossvalidation. We should discuss, whether it makes sense to change this here.